src/OrbitGl/ScopeTree.h

```diff
@@ -36,8 +38,8 @@ class ScopeNode {
 
   [[nodiscard]] ScopeNode* GetLastChildBeforeOrAtTime(uint64_t time) const;
   [[nodiscard]] std::vector<ScopeNode*> GetChildrenInRange(uint64_t start, uint64_t end) const;
-  [[nodiscard]] std::map<uint64_t, ScopeNode*>& GetChildrenByStartTime() {
-    return children_by_start_time_;
+  [[nodiscard]] absl::btree_map<uint64_t, ScopeNode*>& GetChildrenByStartTime() const {
```

Comment:
const method should return const reference, otherwise it is misleading, if caller modifies the map than method is not const.

Reply:
Added const.